doc/articles/create-an-app-rider.md

```diff
@@ -12,17 +12,32 @@ uid: Uno.GettingStarted.CreateAnApp.Rider
 
 ## Create the App
 
-Creating an Uno Platform project is done [using dotnet new](xref:Uno.GetStarted.dotnet-new) by following these steps:
+Creating an Uno Platform project is done [using dotnet new](xref:Uno.GetStarted.dotnet-new) and Uno Platform Online Wizard by following these steps:
+
+1. Open a browser and navigate to the online [App Wizard](https://aka.platform.uno/app-wizard)
+1. Configure your new project by providing a project name, then click **Start**
+
+    ![A screen showing how to choose a solution name](Assets/quick-start/live-wizard-01-choose-name.png)
+
+1. Choose a template to build your application
+
+    ![A screen showing how to choose template for the new app](Assets/quick-start/live-wizard-02-select-preset.png)
+
+    > [!TIP]
+    > For a detailed overview of the Uno Platform project template wizard and all its options, see [this guide](xref:Uno.GettingStarted.UsingWizard).
+
+1. Click the **Create** button on the top right corner, then click the **Copy** button
+
+    ![A screen showing the dotnet new command to create the new app](Assets/quick-start/live-wizard-03-create-app.png)
 
 1. In your terminal, navigate to the folder that will contains your new app.
-1. Create a new project:  
+1. Create a new project by pasting the command that was previously generated in the App Wizard.
+1. Open the solution in Rider, you should now have a folder structure that looks like this:
 
-    ```bash
-    dotnet new unoapp --preset=blank -tfm net7.0 -o MyApp
-    ```
+    ![A screen showing the structure of the solution in Rider](Assets/quick-start/rider-folder-structure.png)
 
-    You should now have a folder structure that looks like this:  
-    ![rider-folder-structure](Assets/quick-start/rider-folder-structure.png)
+> [!TIP]
+> If you are not able to run the online App Wizard, you can explore the [`dotnet new` template](xref:Uno.GetStarted.dotnet-new) directly in the CLI.
```

Comment:
I guess we may want to align on whether we use "online app wizard", "online wizard", or "live wizard" and keep that consistent when we refer to it.